### DISCUSSION TITLE
re-consume messages within the specified range

### DISCUSSION THREAD

**Discussion by Wzy19930507** [spring-projects/spring-kafka#2854] [Q&A] [answered]
hi, when lost the message, i want re-consumption messages within the specified range(like offset 2000 ~ 2100) in runtime.
Are there any good solutions? thanks

**Comment by garyrussell** [comment, marked as answer]
See the documentation about performing seeks (during initialization, or at runtime).

https://docs.spring.io/spring-kafka/docs/current/reference/html/#seek

**Reply by garyrussell** [reply]
But you shouldn't "lose" messages, if you have a suitably configured error handler.

**Reply by Wzy19930507** [reply]
Thank you, it's helpful.
